controllers/dopplersecret_controller_secrets.go

```diff
@@ -83,7 +79,10 @@ func (r *DopplerSecretReconciler) GetReferencedSecret(ctx context.Context, ref s
 
 // GetDopplerToken gets the Doppler Service Token referenced by the DopplerSecret
 func (r *DopplerSecretReconciler) GetDopplerToken(ctx context.Context, dopplerSecret secretsv1alpha1.DopplerSecret) (string, error) {
-	tokenSecret, err := r.GetReferencedSecret(ctx, dopplerSecret.Spec.TokenSecretRef)
+	tokenSecret, err := r.GetReferencedSecret(ctx, types.NamespacedName{
```

Comment:
nit:  it's a little clunky defining the struct in the function call.  I'd just pull it out as a variable on the line above and pass it in.